cfme/tests/services/test_ansible_workflow_servicecatalogs.py

```diff
@@ -69,6 +76,7 @@ def test_tower_workflow_item(appliance, ansible_workflow_catitem, workflow_type,
     )
 
 
+@pytest.mark.uncollectif(versioncheck, reason='API V2 not supported since Tower 3.6.')
```

Comment:
I think this should state `V1 not supported`?

Reply:
Yes